features/environment.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2013-2016 BalaBit
+# Copyright (c) 2013-2018 BalaBit
```

Comment:
AFAIK the correct spelling for the 2nd b has become lowercase a few years ago.

Reply:
Ok, done